rmgpy/data/kinetics/family.py

```diff
@@ -4239,14 +4239,14 @@ def fix_labels_mol(mol, root_labels):
                 rrev.is_forward = False
 
                 if estimate_thermo:
-                    for r in rrev.reactants:
-                        if r.thermo is None:
-                            therm_spc = deepcopy(r)
+                    for rev_react in rrev.reactants:
+                        if rev_react.thermo is None:
+                            therm_spc = deepcopy(rev_react)
```

Comment:
Can we avoid this deepcopy the same way we did in #2153 ?

Reply:
Yeah sure!